app/experimenter/experiments/api/v5/mutation.py

```diff
@@ -1,19 +1,83 @@
 import graphene
-from django.db.models import Field
-from graphene_django.forms.mutation import DjangoModelFormMutation
 
-from experimenter.experiments.api.v5.forms import CreateExperimentForm
+from experimenter.experiments.api.v5.inputs import (
+    CreateExperimentInput,
+    UpdateExperimentInput,
+)
+from experimenter.experiments.api.v5.serializers import NimbusExperimentSerializer
 from experimenter.experiments.api.v5.types import NimbusExperimentType
+from experimenter.experiments.models import NimbusExperiment
 
 
-class CreateExperimentMutation(DjangoModelFormMutation):
-    experiment = Field(NimbusExperimentType)
+class ObjectField(graphene.Scalar):
+    """Utilized to serialize out Serializer errors"""
 
-    class Meta:
-        form_class = CreateExperimentForm
+    @staticmethod
+    def serialize(dt):
+        return dt
+
+
+class CreateExperiment(graphene.Mutation):
+    clientMutationId = graphene.String()
+    nimbus_experiment = graphene.Field(NimbusExperimentType)
+    message = ObjectField()
+    status = graphene.Int()
+
+    class Arguments:
+        input = CreateExperimentInput(required=True)
+
+    @classmethod
+    def mutate(cls, root, info, input: CreateExperimentInput):
+        clientMutationId = input.clientMutationId
+        del input.clientMutationId
```

Comment:
What does mutation id do?

Reply:
Also what happens if you don't delete it out of the input and just pass it to the serializer?

Reply:
Ah, it doesn't apparently care, so I'll stop doing that.